udata/core/dataservices/rdf.py

```diff
@@ -116,10 +117,10 @@ def dataservice_to_rdf(dataservice: Dataservice, graph=None):
     d.set(DCT.issued, Literal(dataservice.created_at))
 
     if dataservice.base_api_url:
-        d.set(DCAT.endpointURL, Literal(dataservice.base_api_url))
+        d.set(DCAT.endpointURL, URIRef(dataservice.base_api_url))
```

Comment:
dataservice endpointURL and endpointDescription are actually URLs and are now exposed as URIRef (same as other URLs in RDF, ex accessUrl and downloadUrl)